docs/database-classes.md

```diff
@@ -279,6 +347,12 @@ transactor.Transact(db =>
 
 ## Limitations
 
```

Comment:
You use "have to", "may not" and "shall" to describe necessities here. Let's just use "must", since it's what most specifications use anyways.